tests/publiccloud/instance_overview.pm

```diff
@@ -14,7 +14,7 @@
 #
 # Maintainer: Pavel Dostal <pdostal@suse.cz>
 
-use base 'consoletest';
+use base "opensusebasetest";
```

Comment:
Is this really neede Felix? The `consoletest` is based on `opensusebasetest` but it has public cloud specific cleaning code in `post_fail_hook()`.

Reply:
I though this is needed for `tar_and_upload`. Will try to switch it back.